hamilton/function_modifiers/configuration.py

```diff
@@ -175,6 +175,10 @@ def _get_function_name(self, fn: Callable) -> str:
     def resolve(self, fn, config: Dict[str, Any]) -> Callable:
```

Comment:
Hmm, why do we have this? Sticking it in tags feels strange. Also, there are cases in which this will have *every* config item (E.G. with `@config.when(lambda x....)` -- the function approach)